aiogram/filters/command.py

```diff
@@ -50,7 +50,7 @@ def __init__(
         self,
         *values: CommandPatternType,
         commands: Optional[Union[Sequence[CommandPatternType], CommandPatternType]] = None,
-        prefix: str = "/",
+        prefix: str = "/!",
```

Comment:
We shouldn't edit default prefix

Reply:
My bad, I forgot to change back after testing.